src/accounting.py

```diff
@@ -972,7 +972,7 @@ def add_proxy(self, author: AccountId, account: Account, proxied_account: Accoun
             self.get_account_id(proxied_account))
         return result
 
-    def delete_account(self, author: Account, account: AccountId):
+    def delete_account(self, author: AccountId, account: AccountId):
```

Comment:
This, as far as I'm aware, is incorrect?

Reply:
I don't really see a reason for this change, maybe as a patch to the master directly but it's not really suitable for what I'm PRing the upstream for.